operator/gitops/argocd/pipeline-service/tekton-results/overlays/dev/minio-user-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  S3_ACCESS_KEY_ID: bWluaW8=
+  S3_SECRET_ACCESS_KEY: bWluaW8xMjM=
```

Comment:
you need to move this file to Bitwarden and then get the secrets from Bitwarden in the shell script before it is used.